api/app/controllers/mno_enterprise/jpi/v1/admin/app_answers_controller.rb

```diff
@@ -14,13 +14,14 @@ def create
 
     def app_answer_params
       # for an admin, the organization does not matter
-      organization_id = current_user.organizations.first.id
+      orga_relation = current_user.orga_relations.first
       params.require(:app_answer).permit(:description)
-        .merge(user_id: current_user.id, question_id: parent.id, organization_id: organization_id, app_id: parent.app_id)
+        .merge(reviewer_id: orga_relation.id, reviewer_type: 'OrgaRelation',
+               parent_id: parent.id, reviewable_id: parent.reviewable_id, reviewable_type: 'App')
```

Comment:
Yes it was. And I don't think it was a good idea. I think it is better to have a clean ReviewResource on the MnoHub side.

![image](https://user-images.githubusercontent.com/4570217/27870679-6ed3e07e-619b-11e7-894f-89d30aaab79f.png)